manifests/olm-catalog/4.2/nfd.v4.2.0.clusterserviceversion.yaml

```diff
@@ -1,19 +1,38 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
+  name: nfd.v4.2.0
+  namespace: placeholder
   annotations:
     capabilities: Basic Install
-    categories: "Configuration"
-    description: This software enables node feature discovery for Kubernetes. It detects hardware features available on each node in a Kubernetes cluster, and advertises those features using node labels.
+    categories: "Database"
```

Comment:
Where are these annotations used, and in what capacity?

Reply:
Just for filtering in the operatorhub ui.